scene/resources/font.cpp

```diff
@@ -178,6 +178,7 @@ PoolVector<int> BitmapFont::_get_kernings() const {
 
 void BitmapFont::_set_textures(const Vector<Variant> &p_textures) {
 
+    textures.clear();
```

Comment:
Indentation is wrong, it should be with tabs instead of spaces. Could you amend the commit to fix that? (See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase))

Otherwise the fix looks correct.

Reply:
Thank you. Should be fixed now.